modular/approver/approve_task.go

```diff
@@ -14,8 +15,9 @@ import (
 )
 
 var (
-	ErrDanglingPointer    = gfsperrors.Register(module.ApprovalModularName, http.StatusBadRequest, 10001, "OoooH.... request lost")
-	ErrExceedBucketNumber = gfsperrors.Register(module.ApprovalModularName, http.StatusNotAcceptable, 10002, "account buckets exceed the limit")
+	ErrDanglingPointer     = gfsperrors.Register(module.ApprovalModularName, http.StatusBadRequest, 10001, "OoooH.... request lost")
```

Comment:
Can you also add them into document at https://docs.bnbchain.org/greenfield-docs/docs/api/storage-provider-rest/sp_response#list-of-error-codes